ci/deps/azure-windows-39.yaml

```diff
@@ -20,20 +20,20 @@ dependencies:
   - html5lib
   - jinja2
   - lxml
-  - matplotlib=2.2.*
+  - matplotlib
   - moto>=1.3.14
   - flask
   - numexpr
-  - numpy=1.17.*
+  - numpy
   - openpyxl
-  - pyarrow=0.17.0
+  # - pyarrow
```

Comment:
There are 17 failures in Python 3.9 Windows build relate to `pyarrow`.

Most of them are:
`pyarrow.lib.ArrowNotImplementedError: Support for codec 'snappy' not built`

Stop testing for now. will open a new issue for this.

Reply:
great can you link the issue here